tools/gen-ssl-dev-certs.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+mkdir -p target/
+openssl req -x509 -newkey rsa:2048 \
+  -keyout target/dev.key.pem \
+  -out target/dev.cert.pem \
+  -sha256 -days 365 \
+  -nodes \
+  -subj "/C=XX/ST=Dev/O=Dev/OU=Dev/CN=webknossos.local"
```

Comment:
:thinking: As this is used for the proxy anyway and there is a tools/proxy dir, maybe move that file here?